safe/definitions/hazard_classifications.py

```diff
@@ -490,7 +749,7 @@
             'affected': True,
             'description': tr('Water above 8.0m.'),
             'string_defaults': [],
-            'displacement_rate': 0.0,
+            'displacement_rate': 1.0,
```

Comment:
1.0 means 100%. And I already made a PR https://github.com/inasafe/inasafe/pull/3782 should I cancel mine ?